pkg/config/config.go

```diff
@@ -52,6 +53,17 @@ type BackupS3 struct {
 	Region  string `yaml:"region"`
 }
 
+type BackupScp struct {
+	Host       string `yaml:"host"`
+	Port       string `yaml:"port"`
+	User       string `yaml:"user"`
+	Path       string `yaml:"path"`
+	SshKey     string `yaml:"sshKey"`
```

Comment:
It prefer to add `sshKeyPath` option and key file setup become out of action's scope.
Users may be add raw key onto yaml and need encrypt it if path option does not exist.

Reply:
Changed to read key paths from environment variables.